include/util.h

```diff
@@ -8,7 +8,7 @@
 #include <xkbcommon/xkbcommon.h>
 
 /**
- * Wrap i into the range [0, max[
+ * Wrap i into the range [0, max]
```

Comment:
I think "[0, max[" is correct.

Unrelated to this PR: I wonder why it's a function actually? Isn't `((i % max) + max) % max` the same as `i % max`?

Reply:
I believe that in C, `%` calculates the remainder. `((i % max) + max) % max` calculates the modulus.

Reply:
`%` _is_ the modulus operator…